src/main/java/com/moorhuhnservice/moorhuhnservice/data/mapper/ConfigurationMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.moorhuhnservice.moorhuhnservice.data.mapper;
+
+import com.moorhuhnservice.moorhuhnservice.data.Configuration;
+import com.moorhuhnservice.moorhuhnservice.data.ConfigurationDTO;
+import java.util.List;
+import java.util.Optional;
+import org.mapstruct.Mapper;
+
+@Mapper(componentModel = "spring")
+public interface ConfigurationMapper {
+  ConfigurationDTO configurationToConfigurationDTO(final Configuration configuration);
+
+  Configuration configurationDTOToConfiguration(final ConfigurationDTO configurationDTO);
+
+  List<ConfigurationDTO> configurationsToConfigurationDTOs(final List<Configuration> configurations);
```

Comment:
Again, would this be possible?
```suggestion
  Collection<ConfigurationDTO> configurationsToConfigurationDTOs(final Collection<Configuration> configurations);
```
Or is it a bad idea as you explicitly require the result to be a list?

Reply:
Can you mention the positive effect? What are the pros?

Reply:
The benefit would be that it would work for `Set`s as well as for `List`s.

Reply:
Bad idea, because it is explicitly a List